iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_queue_types.hpp

```diff
@@ -26,10 +26,10 @@ namespace popo
 struct ChunkTuple
 {
     ChunkTuple() = default;
-    explicit ChunkTuple(relative_ptr<mepoo::ChunkManagement> f_chunk) noexcept;
+    explicit ChunkTuple(rp::RelativePointer<mepoo::ChunkManagement> f_chunk) noexcept;
```

Comment:
You could consider removing the `f_*` in the touched classes if that's not to much effort.